UnityGLTF/Assets/UnityGLTF/Scripts/GLTFSceneExporter.cs

```diff
@@ -179,6 +191,49 @@ private NodeId ExportNode(Transform nodeTransform)
 			return id;
 		}
 
+		private CameraId exportCamera(UnityEngine.Camera unityCamera)
+		{
+			GLTF.Schema.Camera camera = new GLTF.Schema.Camera();
+			//name
+			camera.Name = unityCamera.name;
+			
+			//type
+			bool isOrthographic = unityCamera.orthographic;
+			camera.Type = isOrthographic ? GLTF.Schema.CameraType.orthographic : GLTF.Schema.CameraType.perspective;
+
+			//matrix
+			if (isOrthographic)
+			{
+				GLTF.Schema.CameraOrthographic ortho = new GLTF.Schema.CameraOrthographic();
+				ortho.ZFar = unityCamera.farClipPlane;
+				ortho.ZNear = unityCamera.nearClipPlane;
+				//x and y magnifications: Unity uses a combination of viewport size and origin, as well as "orthographic size"
+				//unsure how to combine these to get the xMag and yMag
+				ortho.XMag = unityCamera.orthographicSize;
```

Comment:
I looked at the raw matrix and xMag is not equal to orthographic size. yMag is though. xMag is being set based on viewport variables. Like I said above, we should just calculate the values specified from the standard based on the camera.perspectiveMatrix

Reply:
Yup, your idea is definitely the safest!! Thanks for this.